app/views/proposal/collection_draft_proposals/show.html.erb

```diff
@@ -22,15 +22,18 @@
         </span>
       <% elsif get_resource.submitted? %>
         <% if @non_nasa_approver %>
-          <%= link_to state_action_button_text('Approve'), "#approve-proposal-modal", class: 'eui-btn--blue display-modal' %>
-          <%= link_to state_action_button_text('Cancel'), "#rescind-submission-modal", class: 'eui-btn--link display-modal' %>
+          <%= link_to state_action_button_text('Approve'), "#approve-submission-modal", class: 'eui-btn--blue display-modal' %>
+          <%= link_to state_action_button_text('Reject'), "#reject-submission-modal", class: 'eui-btn--red display-modal' %>
           <%= render partial: 'proposal/collection_draft_proposals/proposal_partials/approve_modal' %>
-          <%= render partial: 'proposal/collection_draft_proposals/proposal_partials/rescind_modal' %>
-        <% else %>
-          <%= link_to state_action_button_text('Cancel'), "#rescind-submission-modal", class: 'eui-btn--blue display-modal' %>
-          <%= render partial: 'proposal/collection_draft_proposals/proposal_partials/rescind_modal' %>
+          <%= render partial: 'proposal/collection_draft_proposals/proposal_partials/reject_modal' %>
         <% end %>
+        <%= link_to state_action_button_text('Cancel'), "#rescind-submission-modal", class: 'eui-btn--blue display-modal' %>
```

Comment:
This is what I was asking about; should this be a button as an approver?

Reply:
I checked with the PO, she likes it this way for now.

As an aside, she is thinking through if all of the action buttons for the User workflow should be present all the time, with some being disabled depending on the state. You may want to weigh in if you have an opinion about it.